libs/Model/Model.cpp

```diff
@@ -1,70 +1,59 @@
 #include "Model.h"
 #include <string>
 #include <cassert>
+#include "ShaderProgram.h"
 
-
-Model::Model():m_position(0),m_orientation(glm::quat()),m_scale(1), m_bounding_volumen_radius(1)
+Model::Model() : m_position(0), m_orientation(glm::quat()), m_scale(1), m_bounding_volumen_radius(1)
 {
-
 }
 
-
 Model::~Model()
 {
 	delete m_Obj_Model;
-	for (int i = 0 ; i < m_textures.size(); ++i)
-	{
-		delete m_textures[i];
-	}
-		
+
+	delete m_texture;
 }
 
-VertexBuffer& Model::objModel() const
+VertexBuffer &Model::objModel() const
 {
 	return *m_Obj_Model;
 }
 
-void Model::setObjModel(VertexBuffer* val)
+void Model::setObjModel(VertexBuffer *val)
```

Comment:
is there a convention around when a method is snake case vs camel case? I see both in the codebase

Reply:
This is my own external library to load models. It's not part of the volume viewer base code.